esp/clients/wsdfuaccess/wsdfuaccess.cpp

```diff
@@ -608,6 +608,8 @@ class DFUAccessTests : public CppUnit::TestFixture
 protected:
     void testStartServer()
     {
+        setBaseDirectory(getenv("TMPDIR"), 0, DFD_OSdefault);
```

Comment:
This isn't set on windows, so would cause this function to throw an exception.  It would make sense for the logic in eclbundle.cpp(~1040) to be moved into jlib and reused here.

Reply:
Done